config/secrets.yml

```diff
@@ -19,6 +19,9 @@
 
 development:
   secret_key_base: 451f71b291bfde2c7dd465a561820741c3068583a50ffc466643cbae9b705320a28fd5adddea4fbbf7cc1d3c7396cebd0dfecb685f63699bc0ae76f2803a2ea1
+  mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
```

Comment:
be careful here, you are adding this keys only in development. You will probably need this in production too